src/rtsp/stream_decoding.md

```diff
@@ -0,0 +1,126 @@
+# Actually using the stream
```

Comment:
I would favor splitting this into at least two parts:
1. Converting to WebRTC
2. Converting to HLS

In case of HLS we can write it's pretty simply so the chapter might be very short

If there are some common parts we can extract them into one more section.

Reply:
The thing is, I think both of the issues we write about (keyframes and stream params) pertain to WebRTC as well as HLS. Right now, I really can't see a way to split the content. If you really want to do it, please outline the proposed structure of the chapter in more detail.

I added a sentence about the issues being relevant for both conversions to the foreword of this chapter.